src/clusterfuzz/_internal/platforms/android/flash.py

```diff
@@ -64,7 +64,9 @@ def download_latest_build(build_info, image_regexes, image_directory):
   # download it.
   build_id = build_info['bid']
   target = build_info['target']
+  logs.log_warn('target stored in current build_info: %s.' % target)
```

Comment:
I think we just want logs.log for all these logs instead of logs.log_warn

Reply:
+1

Reply:
Done